assets/witnet/addresses.js

```diff
@@ -1,5 +1,7 @@
-
+const { merge } = require("lodash")
```

Comment:
We are using `lodash` but it doesn't appear as a dependency in the `package.json`

Reply:
it's imported from witnet-solidity > witnet-utils

> took note and just changed witnet-utils package.json to import lodash.merge instead of lodash